### DISCUSSION TITLE
License Management & Advanced Security

### DISCUSSION THREAD

**Discussion by Atlas-LinhVN** [community/community#135751] [Enterprise]
### Select Topic Area

Question

### Body

Hi guys,

I, Owner role, currently have 1 Organization and 1 Team with 6 members, 4/6 members already have Pro licenses. Can you please advise the minimum license & type for the Owner to have Advance Security add-on to scan the code created by the members.

My purposes are:

Managing the team that members have Pro licenses

All codes created by members must be scanned (vulnerability, security)

Owner
1 x Enterprise license
1 x Advanced security license
Enterprise license is only needed for the Owner?
Question: Can the Owner use Advanced Security to scan the code created by all members?

Team: members x Pro license.
Question: Pro license is enough for my organization?
Many thanks

**Comment by ebndev** [comment]
Hi @Atlas-LinhVN, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Enterprise` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

**Reply by Atlas-LinhVN** [reply]
No one from answered my question.

Any Github support team can answer my question?

**Comment by Atlas-LinhVN** [comment]
Hi team,

Please advise me the below. Thanks

"I, Owner role, currently have 1 Organization and 1 Team with 6 members, 4/6 members already have Pro licenses. Can you please advise the minimum license & type for the Owner to have Advance Security add-on to scan the code created by the members.

My purposes are:

Managing the team that members have Pro licenses

All codes created by members must be scanned (vulnerability, security)

Owner
1 x Enterprise license
1 x Advanced security license
Enterprise license is only needed for the Owner?
Question: Can the Owner use Advanced Security to scan the code created by all members?

Team: members x Pro license.
Question: Pro license is enough for my organization?"

**Reply by queenofcorgis** [reply]
@Atlas-LinhVN thanks for asking this and your patience. I'm going to highlight this in some content going out next week so your post will be more likely to get an answer!
